src/LaunchDarkly.ClientSdk/Integrations/TestData.cs

```diff
@@ -596,7 +596,7 @@ internal ItemDescriptor CreateFlag(int version, Context context)
                 }
                 int variation;
                 if (!_variationByContextKey.TryGetValue(context.Kind, out var keys) ||
-                    !keys.TryGetValue(context.Key, out variation))
+                    !keys.TryGetValue(context.FullyQualifiedKey, out variation))
```

Comment:
I am trying to map this to my mental model. I think this would be key, because these are per kind.

Reply:
Done